.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What's going on here?

Reply:
This is VS code launch settings for csc.